### ISSUE TITLE
Dates should be expressed in ISO-8601 format

### ISSUE THREAD

**Issue by b-jazz** [osmlab/changeset-map#166]
* **I'm submitting a ... **

  - [X ] bug report
  - [ ] ~~feature request~~ 
  - [ ] support / question

## *Brief Description*

When entering dates in the filter for thanksgiving to christmas, 11/23/2017 -> 12/25/2017 doesn't work because it expects a format that is specific to a particular region of the world that I'm not familiar with. It would be much better if YYYY-MM-DD format was used for the dates since that is a clear, international standard for representing dates.

## How do we replicate the issue ?

1.
2.
3.
4.

**Comment by willemarcel** [comment]
Hi @b-jazz 

Thank you for your report! Are you trying to filter the edits using [OSMCha](http://osmcha.mapbox.com/), right?

**Comment by batpad** [comment]
Thanks for the report @b-jazz -- @willemarcel the date format seems to be defined and hard-coded here: https://github.com/mapbox/osmcha-frontend/blob/master/src/components/filters/date.js#L50

I would definitely be in favour of either adapting to user's locale or following a more standard YYYY-MM-DD. [Obligatory xkcd](https://xkcd.com/1179/)

**Comment by b-jazz** [comment]
@illemarcel that's correct, on the filter page. @batpad thanks for clarifying and including the perfect XKCD.
